packages/tab/src/react/__stories__/index.story.tsx

```diff
@@ -1,14 +1,13 @@
-import PropTypes from 'prop-types'
-import React from 'react'
+import * as React from 'react'
 import { storiesOf } from '@storybook/react'
 
 import Tab from '../index.js'
 
-function randomIntBetween(min, max) {
+function randomIntBetween(min: number, max: number) {
```

Comment:
gettin fancy!

Reply:
I set `strict` in the `tsconfig`, so no implicit `any`s!